### PR TITLE
i18n(it): Add `install/manual.mdx`

### PR DESCRIPTION
**Pull request by milovangudelj**
#### What kind of changes does this PR include?

- Translated content

#### Description

This PR adds the Italian translation of the `install/manual.mdx` page.

### CODE REVIEW

**Comment by milovangudelj** [comment]
@ematipico when you get a chance, could you take a look at this? Thank you! 😁

File: src/content/docs/it/install/manual.mdx [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+  ```
+  </Fragment>
+  <Fragment slot="yarn">
+  ```shell
+  yarn add astro
+  ```
+  </Fragment>
+</PackageManagerTabs>
+
+Poi, sostituisci qualunque sezione segnaposto "scripts" all'interno del tuo file `package.json` con il seguente codice:
```
Comment (ematipico) [inline src/content/docs/it/install/manual.mdx:83]: "segnaposto"? It seems out of place 🤔

File: src/content/docs/it/install/manual.mdx [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+
+Nel tuo editor di testo, crea un nuovo file nella tua cartella al percorso `src/pages/index.astro`. Questa sarà la tua prima pagina Astro nel progetto.
+
+Per questa guida, copia e incolla il seguente blocco di codice (inclusi i tre trattini `---`) all'interno del tuo nuovo file:
+
+```astro title="src/pages/index.astro"
+---
+// Benvenuto/a ad Astro! Tutto ciò che è delimitato da questi tre trattini
+// è il tuo "avantesto del componente". Non viene mai eseguito nel browser.
+console.log('Questo viene eseguito nel tuo terminale, non nel browser!');
```
Comment (ematipico) [inline src/content/docs/it/install/manual.mdx:107]:
```suggestion
console.log('Questo codice viene eseguito nel tuo terminale, non nel browser!');
```

File: src/content/docs/it/install/manual.mdx [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+  </body>
+</html>
+<style>
+  h1 {
+    color: orange;
+  }
+</style>
+```
+
+## 4. Crea il tuo primo asset statico
```
Comment (ematipico) [inline src/content/docs/it/install/manual.mdx:123]: Do you think we can translate "asset" with "risorsa"? Just asking

**Review by ematipico — APPROVED** [review]
Looks good to me!

File: src/content/docs/it/install/manual.mdx (hunk 1 above)
Comment (milovangudelj) [reply src/content/docs/it/install/manual.mdx:83]: Yeah, it's not the most elegant solution but I was going for a literal translation. I'll think of a better one as soon as I get back home.

File: src/content/docs/it/install/manual.mdx (hunk 3 above)
Comment (milovangudelj) [reply src/content/docs/it/install/manual.mdx:123]: Yeah it's possible 👍🏻
I left it there cause I've heard it being used in a lot of places. I can change it if you think it's better

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks for the great work 🚀

**Pull request merged**